docs/MAINTAINING.md

```diff
@@ -63,3 +65,15 @@ It also gives us a way to separate the issue triage from bigger-picture, long-te
 `<major.minor.patch>`
 
 The code-server project follows traditional [semantic versioning](ttps://semver.org/), with the objective of minimizing major changes that break backward compatibility. We increment the patch level for all releases, except when the upstream Visual Studio Code project increments its minor version or we change the plugin API in a backward-compatible manner. In those cases, we increment the minor version rather than the patch level.
+
+## Release
+
+## Release Manager Rotation
```

Comment:
Might want to adjust heading depth here.

Reply:
Fixed!